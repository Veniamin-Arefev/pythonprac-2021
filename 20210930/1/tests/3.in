12, 4, 3, 86, 27, 78, 88, 42, 63, 33, 72, 22, 20, 60, 84, 90, 40, 36, 90, 17, 27, 80, 47, 3, 59, 22, 96, 35, 13, 99, 40, 76, 59, 11, 81, 26, 89, 9, 88, 80, 37, 86, 11, 17, 29, 82, 8, 31, 68, 45, 85, 8, 76, 79, 10, 85, 51, 97, 43, 82, 48, 32, 68, 70, 45, 17, 35, 77, 96, 55, 88, 59, 79, 72, 84, 91, 95, 63, 7, 28, 44, 64, 89, 30, 75, 8, 5, 56, 72, 92, 99, 1, 8, 52, 69, 51, 22, 8, 20, 81