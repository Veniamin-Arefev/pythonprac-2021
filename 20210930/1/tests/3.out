[17, 60, 27, 27, 78, 88, 42, 63, 33, 72, 22, 12, 4, 84, 3, 86, 36, 90, 90, 76, 17, 47, 3, 59, 22, 96, 35, 13, 99, 40, 40, 59, 11, 81, 26, 89, 9, 88, 80, 80, 10, 70, 30, 29, 82, 8, 31, 68, 45, 85, 8, 76, 79, 37, 85, 51, 97, 43, 82, 48, 32, 68, 86, 45, 17, 35, 77, 96, 55, 88, 59, 79, 72, 84, 91, 95, 63, 7, 28, 44, 64, 89, 11, 75, 8, 5, 56, 72, 92, 99, 1, 8, 52, 69, 51, 22, 8, 20, 20, 81]
